user-guide/Cloud_Platform/TeamsBot/DataMiner_Teams_bot.md

```diff
@@ -122,15 +122,18 @@ They support the following output:
 
 - Adaptive card body elements.
 
-- JSON, which is displayed as plain text by the bot.
+- JSON, which is displayed as plain text.
 
 For examples, see [Custom Command Examples](https://github.com/SkylineCommunications/ChatOps-Extensions/tree/main/CustomCommandExamples) on GitHub.
 
+> [!NOTE] Any output cannot exceed the limited size of 35KB.
```

Comment:
The text of a note should always start on the next line:

```
> [!NOTE]
> The size of output is limited to a maximum of 35 KB  
```

Reply:
```suggestion
> [!NOTE]
> The size of output is limited to a maximum of 35 KB.
```